data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```diff
@@ -36,6 +36,11 @@
     "^corda.ledger.keys.[0-9]+.signature.spec$": {
       "description": "Optional. The ledger public key signature spec. Can be multiple specified by index.",
       "type": "string"
+    },
+    "(?=^(?!corda\\.).+)(?=^(?=.{1,128}$)[a-zA-Z0-9]*[^$%^&*;:,<>?()\"']*$)": {
```

Comment:
Maybe we should be more restrictive so that it is something like `[a-zA-Z0-9\.]*` instead of `[a-zA-Z0-9]*[^$%^&*;:,<>?()\"']*`

I tested out a couple keys and these are all valid with this regex
```
my.value=
my.value+
my.value_
my.value-
my.value#
my.value!
my.value@
my.value€
my.value|
my.value\
```